lib/pubsub/topic.js

```diff
@@ -144,11 +147,17 @@ Topic.prototype.publish = function(messages, callback) {
   callback = callback || util.noop;
 
   var body = {
-    topic: this.name,
     messages: messages.map(Topic.formatMessage_)
   };
 
-  this.makeReq_('POST', 'topics/publishBatch', null, body, callback);
+  var path = this.name + ':publish';
```

Comment:
Yeah it's a little different than what we are used to seeing but it's right. See [here](https://cloud.google.com/pubsub/reference/rest/v1beta2/projects/topics/publish). I believe pubsub can get away with this different URL path because they are running their API out of https://pubsub.googleapis.com/ and not simply https://googleapis.com/ like most other APIs.